DP02_17_Galaxy_Photometry.ipynb

```diff
@@ -0,0 +1,1077 @@
+{
```

Comment:
<p>Thanks for making this notebook! Could you update the header part based on the new RTN045? Here is the new template notebook. <a href="https://github.com/lsst/rtn-045/blob/main/_static/template.ipynb" target="_blank">https://github.com/lsst/rtn-045/blob/main/_static/template.ipynb</a> </p><p>For example, "Targeted learning level: beginner" can be skipped. See <a href="https://rtn-045.lsst.io/#organizational-hierarchy" target="_blank">https://rtn-045.lsst.io/#organizational-hierarchy</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Thanks! I made these changes.</p>

Reply:
Thanks for your comments @shenmingfu  ! I think I have addressed them all now. Please let me know what you think!